ui/src/api.ts

```diff
@@ -1,4 +1,7 @@
 import qs from "qs";
+import { get as getStore, writable } from "svelte/store";
+
+export const backendAddressStore = writable("");
```

Comment:
How about encoding the default address here?

```suggestion
export const backendAddressStore = writable("localhost:17246");
```

Reply:
👍  https://github.com/radicle-dev/radicle-upstream/pull/1492/commits/4420975ab7cfc41d091792d380503bc29ab62e4b